app/src/ui/autocompletion/autocompleting-text-input.tsx

```diff
@@ -155,7 +155,7 @@ export abstract class AutocompletingTextInput<
       : -1
     const rect = element.getBoundingClientRect()
     const popupAbsoluteTop = rect.top + coordinates.top
-    const windowHeight = element.ownerDocument.defaultView.innerHeight
+    const windowHeight = element.ownerDocument!.defaultView!.innerHeight
```

Comment:
:art: How do you feel here about being defensive with these checks, rather than `!` to ignore the warnings?

```ts
if (element.ownerDocument == null) {
  return null
}

if (element.ownerDocument.defaultView == null) {
  return null
}

const windowHeight = element.ownerDocument.defaultView.innerHeight
```

I suggest this because in this method we can return `null` if we're not in a known good state, rather than continuing on and raising an error because we try to read something that isn't defined.

Reply:
ah ok. yeah that's a good point. how about we go a step further and do something like b9b04d1?

Reply:
Works for me 👍